packages/wp-story-editor/src/api/utils/normalizeResourceSizes.js

```diff
@@ -15,38 +15,50 @@
  */
 
 /**
- * External dependencies
+ * @typedef {import('@googleforcreators/types').ResourceSize} ResourceSize
  */
-import type { ResourceSize } from '@googleforcreators/types';
 
-interface ResourceSizeInput {
-  width: number | string;
-  height: number | string;
-  mimeType: string;
-  sourceUrl: string;
-}
+/**
+ * @typedef {Object} PartialResourceSize
+ * @property {string|number} [width] Resource width.
+ * @property {string|number} [height] Resource height.
+ * @property {string} mimeType Mime type.
+ * @property {string} [sourceUrl] URL.
+ */
 
 /**
  * Normalize resource sizes to ensure numerical values for dimensions.
  *
- * @param sizes Sizes.
- * @return Normalized sizes.
+ * Skips invalid sizes lacking either width or height.
+ *
+ * @param {Object.<string, PartialResourceSize>} sizes Sizes.
+ * @return {Object.<string, ResourceSize>} Normalized sizes.
  */
-function normalizeResourceSizes(sizes?: { [key: string]: ResourceSizeInput }): {
-  [key: string]: ResourceSize;
-} {
-  const normalizedSizes: { [key: string]: ResourceSize } = {};
+function normalizeResourceSizes(sizes) {
+  const normalizedSizes = {};
 
   if (!sizes) {
     return normalizedSizes;
   }
 
   for (const size of Object.keys(sizes)) {
     const data = sizes[size];
+
+    if (!data.width || !data.height || !data.sourceUrl) {
```

Comment:
This might help with #12485, because that one looks like an issue with trying to do `replaceAll()` an undefined/null `sourceUrl`